src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -135,6 +151,9 @@ class ResourceLocator extends React.Component<FieldTypeProps<?string>> {
                 </div>
                 {formInspector.id &&
```

Comment:
This causes the elements not to be visible on the add form. You should be able to use the same conditional in the `disabled` property of below buttons.

Reply:
Atm both buttons aren't shown in the add form, which i think is a good thing? 
Imo the refresh url button should be disabled when there's not change to the rpl tag fields
history should only be enabled when there's history, or would that be too much to fetch?

How can I detect change on a particular field? In the addFinishFieldHandler? I tried {formInspector.formStore.isFieldModified(dataPath)} but that doesn't seem to do the trick.

Reply:
I would say the difference between the history and the regenerate button is that the history cannot have any values in the add form, therefore I think it makes sense to be hidden. But at the same time the regenerate button might be of value also in the add form.

If you edit the URL field directly, the form does not update its value anymore, even if one of the fields with the `sulu.rlp.part` tag is changed. So after having edited the URL directly it would make sense to have the regenerate button also available in the add form.

Reply:
This issue still seems to be open. As explained in my previous comment, the regenerate button makes sense in the add form, while the history does not.